e2e/libs/recurringjob/recurringjob.py

```
@@ -25,8 +25,9 @@ def create(self,
                cron="*/2 * * * *",
                retain=1,
                concurrency=1,
-               labels={}):
-        return self.recurringjob.create(job_name, task, groups, cron, retain, concurrency, labels)
+               labels={},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)